discovery_engine/lib/src/domain/models/source.dart

```diff
@@ -12,6 +12,9 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+import 'dart:convert' show utf8;
+
+import 'package:csv/csv.dart' show CsvToListConverter;
 import 'package:equatable/equatable.dart' show Equatable;
 import 'package:freezed_annotation/freezed_annotation.dart';
 import 'package:fuzzy/fuzzy.dart' show Fuzzy, FuzzyOptions;
```

Comment:
:wrench: ? `just check` complains locally that the _next_ line (which I can't comment on) is unneeded because the protected in `meta.dart` and in `freezed_annotation.dart` is the same. But this is already the way on master :thinking: ?

Anyway we could just fix it here.

Reply:
I just checked it. I'm running the same version.